src/drivers/intel/cavs/hda-dma.c

```diff
@@ -304,12 +304,8 @@ static void hda_dma_post_copy(struct dma *dma, struct hda_chan_data *chan,
 {
```

Comment:
Can you explain why there is no need for this check, now?

Reply:
@dbaluta There has been never really a need. I've just put it here, because I've noticed it during the refactor. Separation from this PR would mean, that I need to wait for this one to be merged.

Reply:
I'm fine with having this change in this PR. No need to create a separate PR. Anyhow, would be nice to see why do you consider this change is needed. I mean why there is no need to call hda_dma_stop. Maybe because the DMA is already stopped?

Reply:
@dbaluta HDA DMA is not driven using interrupts, so there is no need to asynchronously stop the transfer.

Reply:
@tlauda can you add this explanation in the commit message, please?

Reply:
@dbaluta Done.